src/_gettsim/sozialversicherung/kranken/beitrag/arbeitgeberpauschale_geringfügig.yaml

```
@@ -0,0 +1,24 @@
+---
+arbeitgeberpauschale_bei_geringfügiger_beschäftigung:
```

```suggestion
minijob_arbeitgeberpauschale:
```
symmetric to previous comment, just different namespace.